bumble/l2cap.py

```diff
@@ -1342,6 +1345,12 @@ def __str__(self) -> str:
         )
 
 
+# -----------------------------------------------------------------------------
+# Typing
+# -----------------------------------------------------------------------------
+ChannelMode = TypeVar('ChannelMode', Channel, LeConnectionOrientedChannel)
```

Comment:
Maybe we should use this refactor as an opportunity to rename `LeConnectionOrientedChannel` to `LeCreditBasedChannel`?

Reply:
Yes that's the next step